implementations/rust/ockam/ockam/src/unique.rs

```diff
@@ -1,27 +1,23 @@
 use ockam_core::compat::rand::{thread_rng, Rng};
 use ockam_core::compat::string::String;
 
-/// A simple generator for unique, human-readable identifiers suitable
-/// for use in distributed systems.
-pub struct Unique;
+/// A simple generator for unique, human-readable identifiers suitable for use
```

Comment:
In general, using the word "simple" in docs is not ideal. Would it be possible to drop it? Or is it simple in relation to a more complex generator?